.gitignore

```diff
@@ -2,4 +2,7 @@
 *.ipr
 *.iws
 .idea
-.idea/*
\ No newline at end of file
+.idea/*
+build/*
+.gradle/*
+out/*
```

Comment:
That's werid. I am using Idea Ultimate on Mac OS 10.11, if I do not add those lines to my `.gitignore`, contents in those directory will not be ignored automatically. Am I wrong at somewhere? I'm not an experienced use of Idea.
![image](https://cloud.githubusercontent.com/assets/12689835/18408917/9a99373e-776f-11e6-8926-a39fa5c7557b.png)